src/pypit.py

```diff
@@ -207,7 +207,10 @@ def PYPIT(redname, progname=__file__, quick=False, ncpus=1, verbose=1,
                 vrb = int(a)
             elif o in ('-m', '--use_masters'):
                 use_masters=True
-        lnm = os.path.splitext(arg[0])[0] + ".log"
+        splitnm = os.path.splitext(arg[0])
+        if splitnm[1] != '.pypit':
+            initmsgs.error("Bad extension for PYPIT reduction file.  .pypit is required")
```

Comment:
How about:

`initmsgs.error("Bad extension for PYPIT reduction file."+msgs.newline()+".pypit is required")`